backend/src/ee/services/dynamic-secret/dynamic-secret-service.ts

```diff
@@ -282,8 +285,8 @@ export const dynamicSecretServiceFactory = ({
       actorOrgId
     );
     ForbiddenError.from(permission).throwUnlessCan(
-      ProjectPermissionActions.Edit,
-      subject(ProjectPermissionSub.Secrets, { environment: environmentSlug, secretPath: path })
+      ProjectPermissionDynamicSecretActions.Edit,
```

Comment:
shouldn't getting details be `Read`?

Reply:
It should be `Edit`. The is not normal read. This is detail read of dynamic secret configuration. Meaning this contains decrypted root configuration value